index.html

```diff
@@ -285,6 +285,11 @@ <h1 class="white-header">FOSSASIA on social media</h1>
                         </div>
                     </div>
                 </div>
+				<!--Flickr-->
+				<div class="row">
+					<div id="flickrembed"></div>
+					<small style="display: block; text-align: center; margin: 0 auto;"><a href="//www.flickr.com/photos/fossasia">Fossasia on Flickr</a>.</small>
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/index.html
+++ b/index.html
@@ -288,7 +288,7 @@
 				<!--Flickr-->
 				<div class="row">
 					<div id="flickrembed"></div>
-					<small style="display: block; text-align: center; margin: 0 auto;"><a href="//www.flickr.com/photos/fossasia">Fossasia on Flickr</a>.</small>
+                    <small style="display: block; text-align: center; margin: 0 auto;"><a href="//www.flickr.com/photos/fossasia">Fossasia on Flickr</a>.</small>
 				</div>
             </div>
         </div>
```